tests/telemetry/test_series.py

```diff
@@ -0,0 +1,48 @@
+import time
+
+from ddtrace.internal.telemetry.data.host import get_hostname
+from ddtrace.internal.telemetry.data.metrics import Series
+
+
+def test_series_to_dict():
+    series = Series("test.metric")
+
+    assert series.to_dict() == {
+        "metric": "test.metric",
+        "points": [],
+        "tags": {},
+        "type": Series.COUNT,
+        "common": False,
+        "interval": None,
```

Comment:
I don't see anywhere where we are testing these properties, that is, there are no test cases that create gauges or common metrics, or which have a different interval.

Reply:
I added more comprehensive tests